docs/integrations/README.md

```diff
@@ -65,6 +65,7 @@ Airbyte uses a grading system for connectors to help users understand what to ex
 |[Microsoft Teams](./sources/microsoft-teams.md)| Certified |
 |[Mixpanel](./sources/mixpanel.md)| Beta |
 |[Mongo DB](./sources/mongodb.md)| Alpha |
+|[Mongo DB new](./sources/mongodb-new.md)| Alpha |
```

Comment:
there should be only one mongo db, also it should be in beta

Reply:
Done